apps/demos/project.json

```
@@ -102,6 +111,12 @@
         },
         "zoneless": {
           "buildTarget": "demos:build:zoneless"
+        },
+        "development": {
+          "buildTarget": "demos:build:development",
+          "open": true,
+          "liveReload": true,
+          "watch": true
```

```suggestion
          "open": true
```

liveReload and watch are both true by default, we can omit those. If you like to keep open, it's ok for me

---

@hoebbelsB ok, so im deleting liveReload, and watch